quickstart/prerequisites.md

```diff
@@ -5,37 +5,208 @@
 All worker nodes must satisfy the following requirements:
 
 * **x86-64** CPU cores with SSE4.2 instruction support:
-  * Intel Nehalem processor \(march=nehalem\) and newer
+  * First generation Intel Core i5 and Core i7 (Nehalem microarchitecture) or newer
   * AMD Bulldozer processor and newer
 
 * Linux kernel **5.4 or higher** with the following modules loaded:
   * nvme-tcp
   * ext4 and optionally xfs
 
-## **Mayastor pods (Data Plane/Engine)**
-Each worker node which will host an instance of a Mayastor pod must have the following resources _free and available_ for use by Mayastor:
```

Comment:
why is this direction being removed?  It is important that the nodes have this level of resource _free_ for Mayastor to use.  It is not enough for worker nodes to simply have that much resource - it could be in use by other processes etc.

Reply:
This information has been added under 'Recommended resource requirements' ( Line number 34)